Resources/Prototypes/_NF/Recipes/Cooking/meal_recipes.yml

```diff
@@ -166,6 +166,56 @@
     FoodCucumber: 1
   reagents:
     Saline: 10
+    
+- type: microwaveMealRecipe
+  id: RecipePoachedPears
+  name: poached pears
+  result: FoodMealPoachedPears
+  time: 10
+  solids:
+    FoodBowlBig: 1
+    FoodPear: 3
+    FoodOrange: 2
+  reagents:
+    Wine: 10
+    
+- type: microwaveMealRecipe
+  id: RecipeBellaPears
+  name: bella pears
+  result: FoodMealBellaPears
+  time: 10
+  solids:
+    FoodBowlBig: 1
+    FoodPear: 3
+    FoodSnackChocolateBar: 2
+    FoodLemon: 1
+  reagents:
+    Cream: 5
+
+- type: microwaveMealRecipe
+  id: RecipePearMuffin
+  name: pear muffin
+  result: FoodMealPearMuffin
+  time: 10
+  solids:
+    FoodPear: 1
+    FoodSnackChocolateBar: 1
+  reagents:
+    Flour: 5
+    Oats: 5
+    Sugar: 5
+    
+- type: microwaveMealRecipe
+  id: RecipePearCheeseTart
+  name: pear and goat cheese tart
+  result: FoodMealPearTart
+  time: 10
+  solids:
+    FoodPlateTin: 1
+    FoodDoughPie: 1
+    FoodPear: 3
+    FoodChevre: 1
+    FoodButter: 1
```

Comment:
Most pie recipes have around 3 "solid flavour" ingredients, if you see what I mean. 3 apples, 3 berries, 3 meat, 3 xeno meat, 3 grapes, 3 mimanas, 3 chocolate bars... Comparing the ingredients that go into the pie with the reagent content of the pie, do you think it might make more sense to have 2 pears and 1 chèvre?

Reply:
Fair, done, removed the nutrition value to be the base one, and renamed the ID to FoodTartPearCheese